src/Chainweb/Version.hs

```diff
@@ -736,8 +726,6 @@ slowEpochGuard _ _ = False
 --
 oldTargetGuard :: ChainwebVersion -> BlockHeight -> Bool
 oldTargetGuard Mainnet01 h = h < 452820 -- ~ 2020-04-04T00:00:00Z
-oldTargetGuard Testnet04 h = h < 286352 -- ~ 2020-04-02T03:00:00Z
-oldTargetGuard Development h = h < 360
```

Comment:
This isn't needed any more. We should simplify the logic behind this guard further in a follow up PR, by just skipping target validation for mainnet blocks below above height. Those blocks are grandfathered into the history of mainnet as witnessed by the top-level Merkle hash.